js/src/Bars.ts

```diff
@@ -80,12 +80,12 @@ export class Bars extends Mark {
         const x_scale = this.scales.x, y_scale = this.scales.y;
         this.listenTo(x_scale, "domain_changed", function () {
             if (!this.model.dirty) {
-                this.draw();
+                this.draw(false);
```

Comment:
Is there a reason for removing the animation when the domain changes or is it a leftover from a debug session?

Reply:
I changed this flag to true now so we have an animation consistent with the initial screenshot!